src/app/components/footer/footer-links/footer-links.component.html

```diff
@@ -0,0 +1,17 @@
+<ng-container class="footerLinks" *ngIf="footerLinks.status == 'ok'">
+  <div
+    class="linkBlock"
+    *ngFor="let linkBlock of footerLinks.links; let i = index"
+  >
+    <ng-template [ngIf]="i < 3">
```

Comment:
Could you please explain me why we are using template?
and iterator?

Reply:
Apparently this is not necessary, simple div will do job without  [ngIf]="i < 3"